### ISSUE TITLE
Show custom emoji shortcodes when emoji don't load. 

### ISSUE THREAD

**Issue by Fidgetcetera** [tuskyapp/Tusky#1481]
Due to an instance media outage, a lot of emoji are not loading right now, and it means I'm completely unable to see what emoji people are posting. A blank space just sits where the emoji should be. It'd be nicer if the emoji that aren't loading would display the text shortcodes.

* * * *
- Tusky Version: 8.1
- Android Version: 10
- Android Device: Google pixel 2
- Mastodon instance (if applicable): Monsterpit.net but appears not to be instance specific.

- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by charlag** [comment]
Hi, thanks for reporting. This doesn't seem great, I agree, I need to check if it's possible to fall back to text

**Comment by connyduck** [comment]
Text that jumps around is very weird imho, I would prefer to just not see the emojis.

**Comment by noiob** [comment]
At least some kind of replacement icon while the images are loading would be nice? It's really annoying to have to guess whether a post is empty or if there's an emoji not loading.

Also being able to see the shortcode from any emoji would be really handy, maybe something analog to alt text on desktop?

**Comment by charlag** [comment]
@noiob yeah, but which alternative? I experimented with long-press but it's absolutely not discoverable or convenient.

**Comment by noiob** [comment]
hm, maybe adding an entry to the three-dots menu? it could open a popover that just displays all the emoji and their shortcodes, maybe with an option to copy the shortcodes

**Comment by Sonickyle27** [comment]
In Slide for Reddit, a "Copy text" option is provided for comments in the "three dots" menu for it, that when tapped, shows a popup of the plaintext/markdown of the comment.

If something like this was added for Tusky, not only would it allow users to copy specific parts of statuses rather than all of one when long pressing, but it could also show shortcodes within.
